src/v2/Apps/Artwork/Components/ArtworkMeta.tsx

```diff
@@ -111,6 +125,20 @@ export class ArtworkMeta extends Component<ArtworkMetaProps> {
     )
   }
 
+  renderZendeskScript() {
+    const { artwork } = this.props
+    if (
+      artwork.is_in_auction ||
+      !artwork.is_acquireable ||
+      !artwork.isOfferable ||
+      !artwork.isInquireable
+    )
+      return
```

Comment:
I think the logic might be off. We want to show the widget for BNMO or inquiry works. So maybe something like
```javascript
const { is_in_auction, is_acquireable, isOfferable, isInquireable } = artwork

if (is_in_auction) return
if (!is_acquireable && !isOfferable && !isInquireable) return
```

Reply:
Can't a work just be inquireable and in that case we'd want to display the widget?

Reply:
Yeah so the logic would _not_ early return when `isInquireable` is `true`. Or maybe I miss something here. 🤔

Reply:
No you are right. I totally over looked that.